page-objects/pages/administrationPages/ADMNSTRTNAdministerStudentsPage.js

```diff
@@ -8,15 +8,16 @@ const elementHelpers = require('../../../runtime/helpers/elementHelpers');
 let oldPassword;
 
 const addStudentBtn = "button[data-testid='btn_add_student']";
-const firstNameInput = "input[data-testid='create_student_input_firstname']";
-const lastNameInput = "input[data-testid='create_student_input_lastname']";
-const emailInput = "input[data-testid='create_student_input_email']";
+const firstNameInput = '#create_firstname';
```

Comment:
this could be moved to data-test-ids

Reply:
Okay, I didn't know that this is the best-practice. Thanks for your feedback, I will change that accordingly.